src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -99,6 +100,11 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                         _sourceName = propertyData.IndexerName;
                     }
                 }
+
+                if (this.ReturnsByRefReadonly)
```

Comment:
> if (this.ReturnsByRefReadonly) [](start = 15, length = 31)

It is not clear why is this done conditionally. Also, I think this check doesn't belong in a constructor.